pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -47,7 +47,8 @@ const (
 	CloudName             = "openstack"
 	CloudsSecretKey       = "clouds.yaml"
 	// NOTE(dulek): This one is hardcoded in openshift/installer.
-	InfrastructureCRDName = "cluster"
+	InfrastructureCRDName           = "cluster"
```

Comment:
this needs gofmt I think

Reply:
Actually gofmt put it like that - seems like the comment stops the alignment section.